test/e2e/node/pod_resize.go

```diff
@@ -1408,14 +1441,133 @@ func doPodResizeErrorTests() {
 	}
 }
 
-var _ = SIGDescribe("Pod InPlace Resize Container [Feature:InPlacePodVerticalScaling]", func() {
-	featureGatePostAlpha = true
-	if fs, found := utilfeature.DefaultFeatureGate.DeepCopy().GetAll()[features.InPlacePodVerticalScaling]; found {
-		if fs.PreRelease == featuregate.Alpha {
-			featureGatePostAlpha = false
+func doPodResizeSchedulerTests() {
+	f := framework.NewDefaultFramework("pod-resize-scheduler")
+
+	ginkgo.It("pod-resize-scheduler-testcase1", func() {
+		nodes, _ := f.ClientSet.CoreV1().Nodes().List(context.TODO(), metav1.ListOptions{})
+		nodeNum := len(nodes.Items)
+		if nodeNum > 1 {
+			framework.Logf("There are more than 1 node in the testing cluster, so skip the pod-in-queue-resize-test!")
+			return
 		}
-	}
+
+		ginkgo.By("Checking the node allocatable CPU resources!")
+		node := nodes.Items[0]
+		nodeAllocatableResources := node.Status.Allocatable
+		nodeAllocatableCPU := nodeAllocatableResources.Cpu()
+		nodeAllocatableCPUValue := nodeAllocatableCPU.MilliValue()
+		nodeAllocatableCPUStr := nodeAllocatableCPU.String()
+		framework.Logf("Node allocatable CPU is %s", nodeAllocatableCPUStr)
```

Comment:
Please add nodeName to this log. And we can reduce the number of variables used as well to make the code more compact.

nodeAllocatableCPU = node.Status.Allocatable.Cpu()  {please check if Allocatable can be nil, iirc it should not be)
then
testPodRequest := nodeAllocatableCPU. MilliValue() / 2
or
Logf(" ..." , nodeAllocatableCPU.String()